### PR TITLE
[Term Entry] PHP String Function: str_pad()

### LINKED ISSUE #2434: [Term Entry] PHP String Functions str_pad()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `str_pad()` function.

The entry should go in a new file in `docs/content/php/concepts/string-functions/terms/str-pad/str-pad.md` (Note the path uses dashes, not underscores.)

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
### Description

- Added the str_pad() term for PHP
- Closes #2434 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by KTom101 — CHANGES_REQUESTED** [review]
@phuongtrieu97coder I've done a first review and added some changes to this.

**Review by KTom101 — APPROVED** [review]
@phuongtrieu97coder Thanks for making the changes. All you have to do is wait for another reviewer for a 2nd review.

File: content/php/concepts/string-functions/terms/str-pad/str-pad.md
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/str-pad/str-pad.md:24]:
```suggestion
The `str_pad()` function has two required parameters and two optional parameters:
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@phuongtrieu97coder I did a second review and left a few comments.

**Review by SSwiniarski — APPROVED** [review]
@phuongtrieu97coder I did one more edit. Looks ready to merge.

**Comment by SSwiniarski** [comment]
@phuongtrieu97coder Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/string-functions/str-pad

image

**Pull request merged**
